countries/views/questions.erb

```diff
@@ -0,0 +1,14 @@
+<html>
+    <body>
+        <a href="/questions/new">New</a>
+        <% @questions.each do |q| %>
+            <div>
+                <a href="questions/edit?id=<%= q.id %>">Edit</a>
+                <%= q.question %>
+                <%= q.answer %>
+                <%= q.flag %>
+                <img height="100" src= "<%= q.flag %">
```

Comment:
Again, dumb computers 💻 ...they will get very confused because the closing tag is missing here. Instead of `<img height="100" src= "<%= q.flag %">`, try `<img height="100" src= "<%= q.flag %>">`